jenkins/Jenkinsfile

```
@@ -36,41 +45,79 @@ pipeline {
     timestamps()
   }
   stages {
-    stage('Build') {
-      steps {
-        container('node'){
-          dir("${env.WORKSPACE}/${env.PROJECT}") {
-            echo "Install dependencies"
-            sh 'npm ci --prefer-offline'
+    stage('CI & Audit'){
```

Maybe
```suggestion
    stage('Install & Audit'){
```
would be more accurate? (as CI might be misleading)

---

I'll take your suggestion